TrailBlaze/Networking/Interceptor.h

```diff
@@ -0,0 +1,18 @@
+//
+//  Interceptor.h
+//  TrailBlaze
+//
+//  Created by Adam Issah on 7/20/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "CoreLocation/CoreLocation.h"
+#import "MapKit/MapKit.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Interceptor : NSObject
++ (void) getBestETAPoint: (NSArray *) allPoints interceptorLocation: (CLLocation *) interceptorLocation runnerLocation: (CLLocation *) runnerLocation completion:(void (^)(MKMapItem *bestPoint, NSError *))completion;
```

Comment:
the completion block should specify `NKMapItem * _Nullable bestPoint` since you do return `nil` in some cases

Reply:
and same for `NSError`